modules/tinymce/src/themes/silver/main/ts/api/Options.ts

```diff
@@ -302,6 +307,12 @@ const useBranding = option('branding');
 const getResize = option('resize');
 const getPasteAsText = option('paste_as_text');
 const getSidebarShow = option('sidebar_show');
+const promotionEnabled = option('promotion');
+
+const hasApiKey = (editor: Editor) => {
+  const key = option('api_key')(editor);
```

Comment:
```suggestion
  const key = editor.options.get('api_key');
```
This should just be doing this, either that or the option should be added as a function above.